src/smart-components/user/add-user-to-group/add-user-to-group-wizard.js

```diff
@@ -0,0 +1,251 @@
+import React, { useRef, useContext, useState, useEffect } from 'react';
+import { Modal, Button, ModalVariant } from '@patternfly/react-core';
+import FormRenderer from '@data-driven-forms/react-form-renderer/form-renderer';
+import Pf4FormTemplate from '@data-driven-forms/pf4-component-mapper/form-template';
+import { createRows } from '../../group/group-table-helpers';
+import { applyFiltersToUrl, areFiltersPresentInUrl, syncDefaultFiltersWithUrl } from '../../../helpers/shared/filters';
+import PropTypes from 'prop-types';
+import { Alert } from '@patternfly/react-core';
+import { WarningModal } from '../../common/warningModal';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/';
+import GroupRowWrapper from '../../group/group-row-wrapper';
+import { sortable } from '@patternfly/react-table';
+import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import { shallowEqual, useSelector, useDispatch } from 'react-redux';
+import { createQueryParams } from '../../../helpers/shared/helpers';
+import messages from '../../../Messages';
+import pathnames from '../../../utilities/pathnames';
+import { useIntl } from 'react-intl';
+import { TableToolbarView } from '../../../presentational-components/shared/table-toolbar-view';
+import { useFieldApi } from '@data-driven-forms/react-form-renderer';
+import PermissionsContext from '../../../utilities/permissions-context';
+import { addMembersToGroup, fetchGroups } from '../../../redux/actions/group-actions';
+import { useHistory } from 'react-router-dom';
+import { getUserGroups } from '../../../helpers/group/group-helper';
+
+const FORM_ID = 'add-user-to-group-form';
+
+const GroupTable = (conf) => {
+  const { input, user, setLength } = useFieldApi(conf);
+  const textFilterRef = useRef(null);
+  const { orgAdmin, userAccessAdministrator } = useContext(PermissionsContext);
+  const isAdmin = orgAdmin || userAccessAdministrator;
+  const dispatch = useDispatch();
+  const intl = useIntl();
+  const history = useHistory();
+
+  const columns = [
+    { title: intl.formatMessage(messages.name), key: 'name', transforms: [sortable] },
+    { title: intl.formatMessage(messages.roles) },
+    { title: intl.formatMessage(messages.members) },
+    { title: intl.formatMessage(messages.lastModified), key: 'modified', transforms: [sortable] },
+  ];
+
+  const { groups, meta, filters, isLoading } = useSelector(
+    ({ groupReducer: { groups, isLoading } }) => ({
+      groups: [...(groups?.data?.filter(({ platform_default, admin_default } = {}) => !(platform_default || admin_default)) || [])],
+      meta: groups?.pagination || groups?.meta,
+      filters: groups?.filters || '',
+      isLoading,
+    }),
+    shallowEqual
+  );
+  const [pagination, setPagination] = useState(meta);
+  const [filterValue, setFilterValue] = useState(filters.name || '');
+  const [removeGroupsList, setRemoveGroupsList] = useState([]);
```

Comment:
we should not be able to edit/remove groups from the modal, it is not present in the mockups. Can you please remove the actions cell? There should be only `Name` and `Description`